src/main/java/com/playmonumenta/networkrelay/RemotePlayerManager.java

```diff
@@ -108,100 +113,105 @@ public void broadcast() {
 	private static final Map<String, RemotePlayer> mRemotePlayersByName = new ConcurrentSkipListMap<>();
 	private static final Set<UUID> mVisiblePlayers = new ConcurrentSkipListSet<>();
 
-	protected RemotePlayerManager() {
+	private RemotePlayerManager() {
```

Comment:
We want to prevent more than one instance from being created, even in the plugin itself. Switched back to `getInstance()` style.

Reply:
Would say to move those constants up towards the top of the file, but probably isn't necessary